examples/android/helloworld/app/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.4.1)
+cmake_minimum_required(VERSION 3.8)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
Drive-by: looks like yo could remove this?

Reply:
Is this because gRPC cmakefile has this already?

Reply:
Yes, any target that depends on gRPC++ will inherit the requirement to use C++ 14 or higher.